docs/4-cloud-computing/4.3.5-aci.md

```diff
@@ -77,54 +71,38 @@ az acr create \
 az acr login --name $REGISTRY_NAME
 ```
 
-9. Get the server address of your ACR instance.
-
-```bash
-DOCKER_SERVER=$(az acr show \
-  --name $REGISTRY_NAME \
-  --query loginServer \
-  --output tsv)
-```
+9. Get the server address of your ACR instance (`az acr show -h`) and store it in `DOCKER_SERVER`.
 
 10. Tag the image and push to ACR.
 
-```bash
-docker tag aci-bootcamp $DOCKER_SERVER/aci-bootcamp:v1
-```
-
-```bash
-docker push $DOCKER_SERVER/aci-bootcamp:v1
-```
+?> The tag for the image does not need to be specific, only needs to be consistant
 
 11. Use Azure CLI to List images in your ACR instance and make sure your Docker push was successful.
 
-12. Create a service principles to use with ACR. [Click here](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals) to learn more about service principal objects in Azure AD.
+12. Create a service principles to use with ACR. [Click here](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals) to learn more about service principal objects in Azure AD. You will need to find and set the following environment variables, or locate the values.
 
 ```bash
-SP_NAME="http://YOUR_NAME-devops-bootcamp-acr"
-REGISTRY_ID=$(az acr show --name $REGISTRY_NAME --query id --output tsv)
-DOCKER_PASSWORD=$(az ad sp create-for-rbac --name $SP_NAME --scopes $REGISTRY_ID --role acrpull --query password --output tsv)
-DOCKER_USERNAME=$(az ad sp list --display-name $SP_NAME --query '[0].appId' --output tsv)
+$SP_NAME
+$REGISTRY_ID
+$DOCKER_PASSWORD
+$DOCKER_USERNAME
```

Comment:
This feels like we went from training wheels to "zero help figure it the fuck out" 

We should describe what each piece of information we need is and why. Someone new is not going to see `DOCKER_PASSWORD` and think "oh I need to create an rbac for my service principle scoped to my registry that allows me to pull" they are probably going to enter their password for Docker Hub